core/vm/evm.go

```diff
@@ -139,7 +140,11 @@ func NewEVM(ctx Context, statedb StateDB, chainConfig *params.ChainConfig, vmCon
 		interpreters: make([]Interpreter, 1),
 	}
 
-	evm.interpreters[0] = NewEVMInterpreter(evm, vmConfig)
+	if len(os.Getenv("EVMC_PATH")) != 0 {
```

Comment:
What exactly is geth doing with that list?

Reply:
I tried the command line switch but this is not doable currently (at least not in this PR) because the `VMConfig{}` is created in 100 places usually with default value just because the it is required to create an EVM.

Reply:
@axic it lists the interpreters, so as to be able to find the best format for a given executable. This is how you can mix EVM&EWASM contracts.